src/formatting_stack/project_parsing.clj

```diff
@@ -22,12 +23,21 @@
        (map (speced/fn [^File f]
               (-> f .getCanonicalFile)))))
 
+(speced/defn ^{::speced/spec (spec/coll-of any? :min-count 1)} classpath-directories
```

Comment:
One-off QAing for this defn:

```clj
(let [a (-> (classpath-directories)
            (find-files find/clj)
            (set))
      b (-> (classpath/classpath-directories)
            (find-files find/clj)
            (set))]
  (check! empty? (set/difference b a)))
```

(I considered and discarded adding this piece of logic to the test suite... it wouldn't pass in JDK11, and we don't necessarily have to test against 3rd party bugs anyway)